.changeset/early-trains-run.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-pwa/composables-next": patch
+---
+
+Load shipping address for the order details
```

Comment:
what is exact change in API of composables, something like:
`useSomething` - in `loadOrders` function we're also fetching shipping address